### PR TITLE
python3Packages.mautrix: 0.18.9 -> 0.19.3

### PR DESCRIPTION
**Pull request by NickCao**
Changelog: https://github.com/mautrix/python/releases/tag/v0.19.3

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by NickCao** [comment]
Result of `nixpkgs-review pr 213717` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages failed to build:</summary>
  <ul>
    <li>mautrix-googlechat</li>
    <li>mautrix-signal</li>
    <li>mautrix-telegram</li>
  </ul>
</details>
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>heisenbridge</li>
    <li>mautrix-facebook</li>
    <li>python310Packages.mautrix (python310Packages.mautrix-appservice)</li>
    <li>python311Packages.mautrix (python311Packages.mautrix-appservice)</li>
  </ul>
</details>

**Comment by NickCao** [comment]
Result of `nixpkgs-review pr 213717` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>7 packages built:</summary>
  <ul>
    <li>heisenbridge</li>
    <li>mautrix-facebook</li>
    <li>mautrix-googlechat</li>
    <li>mautrix-signal</li>
    <li>mautrix-telegram</li>
    <li>python310Packages.mautrix (python310Packages.mautrix-appservice)</li>
    <li>python311Packages.mautrix (python311Packages.mautrix-appservice)</li>
  </ul>
</details>

**Review by sumnerevans — APPROVED** [review]
Result of `nixpkgs-review pr 213717` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>7 packages built:</summary>
  <ul>
    <li>heisenbridge</li>
    <li>mautrix-facebook</li>
    <li>mautrix-googlechat</li>
    <li>mautrix-signal</li>
    <li>mautrix-telegram</li>
    <li>python310Packages.mautrix (python310Packages.mautrix-appservice)</li>
    <li>python311Packages.mautrix (python311Packages.mautrix-appservice)</li>
  </ul>
</details>

File: pkgs/development/python-modules/mautrix/default.nix [hunk 1]
```diff
@@ -1,40 +1,67 @@
...
 
   propagatedBuildInputs = [
     aiohttp
+    attrs
+    yarl
+  ];
 
-    # defined in optional-requirements.txt
-    sqlalchemy
-    aiosqlite
```
[the remark this answers is not in the document]
Comment (NickCao) [reply pkgs/development/python-modules/mautrix/default.nix:30]: Just followed the upstream `setup.py`, see https://github.com/mautrix/python/blob/master/setup.py

**Pull request merged**